docs/smart-contracts/creator-tools/ERC20Minter.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: ERC20 Minter
+---
+---
+
+The ERC20 Minter contract allows you to mint 1155 tokens with ERC20 tokens. This contract is deterministically deployed on all chains that we currently support to this address `0x777777E8850d8D6d98De2B5f64fae401F96eFF31`. 
+
+When minting with the ERC20 Minter contract there is a 5% reward that is split amongst the reward recipients in the ERC20 currency. A breakdown of these percentages looks like:
+
+#### ERC20 Mint Reward Percentages
+
+| Recipient                            | Amount                                                                                                                                                        |
+| -------------------------------------| --------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Creator                              | 95% of earnings from sales                                                                                                                                    |
+| Mint Fee                             | 5% of listed purchase price                                                                                                                                   |
```

Comment:
is this for the existing erc20 minter or new one?

Reply:
existing (v1) as we have not released the new version